config.go

```diff
@@ -95,15 +95,29 @@ type Config struct {
 
 	// File logger options
 	FileLogger *FileLoggerConfig `mapstructure:"file_logger_options"`
+
+	// UseLocalTimeEncoder is used to set the encoder to use local time instead of UTC.
+	UseLocalTimeEncoder bool `mapstructure:"use_local_time_encoder"`
+
+	// ShowCaller is used to set the encoder to show the caller.
+	ShowCaller bool `mapstructure:"show_caller"`
 }
 
 // BuildLogger converts config into Zap configuration.
 func (cfg *Config) BuildLogger() (*zap.Logger, error) {
 	var zCfg zap.Config
+	callerKey := zapcore.OmitKey
+	if cfg.ShowCaller {
+		callerKey = "caller"
+	}
 	switch Mode(strings.ToLower(string(cfg.Mode))) {
 	case off, none:
 		return zap.NewNop(), nil
 	case production:
+		encodeTime := utcEpochTimeEncoder
+		if cfg.UseLocalTimeEncoder {
+			encodeTime = localTimeEncoder
+		}
```

Comment:
Local time is often the source of the errors when using different time zones.

Reply:
For projects that operate within a single timezone (which is common for many applications), local time provides more intuitive logging without the complexity of timezone management

Reply:
Yeah, this is fair. I agree with that change, but tests should be added + docs + shorter name in the configuration.